iOS/README.md

```diff
@@ -1,3 +1,22 @@
 # ChatDemo-SwiftUI  
 
-Basic Chat application written with SwiftUI  
+Basic Chat application written with Ditto and SwiftUI/Combine  
+
+1. Clone this repo to a location on your machine and open in Xcode.  
```

Comment:
Is there anything required here for APP_ID/TOKEN with `Env.swift` for publishing to the app store? @kndoshn

Reply:
Yes, I think we should hide the license token and the app ID like other apps: https://www.notion.so/getditto/Environment-Variables-78261e05a2b44a299ee388f06e9ff86a

@magicwave Please feel free to reach out if you have any questions.

Reply:
Ok, I'll implement the Env.swift pattern and update this PR

Reply:
Implemented Env.swift pattern and updated iOS README. 
(Note: rebased onto main and force pushed this PR branch)